### PR TITLE
Cloudwatch: Docs improvements

### PR DESCRIPTION
**Pull request by sunker**
There's quite a lot of repetition from the `what's new in 6.5`, but I think that is fine.

### CODE REVIEW

File: docs/sources/features/datasources/cloudwatch.md [hunk 1]
```diff
@@ -254,6 +254,17 @@ it costs $0.01 per 1,000 GetMetricStatistics or ListMetrics requests. For each q
 issue a GetMetricStatistics request and every time you pick a dimension in the query editor
 Grafana will issue a ListMetrics request.
 
+## Service Quota
+
+AWS defines quotas, or limits, for resources, actions, and items in your AWS account. Depending on the number of queries in your dashboard and the amount of users accessing the dashboard, you may reach the limit for the allowed number of CloudWatch GetMetricData requests per second. Note that quotas are defined per account and per region. If you are using multiple regions or have set up more than one CloudWatch data source to query against multiple accounts, you need to request a quota increase for each account and each region.
```
Comment (marcusolsson) [inline docs/sources/features/datasources/cloudwatch.md]: `you are` -> `you're`  

[Style guide](https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions)

File: docs/sources/features/datasources/cloudwatch.md [hunk 2]
```diff
@@ -254,6 +254,17 @@ it costs $0.01 per 1,000 GetMetricStatistics or ListMetrics requests. For each q
...
+ 
+
+To request a quota increase, visit the [AWS Service Quotas console](https://console.aws.amazon.com/servicequotas/home?r#!/services/monitoring/quotas/L-5E141212).
+
+
```
Comment (marcusolsson) [inline docs/sources/features/datasources/cloudwatch.md]: We should avoid multiple newlines between paragraphs. Paragraphs should be seperated by one blank line:

```
To request a quota increase, visit the [AWS Service Quotas console](https://console.aws.amazon.com/servicequotas/home?r#!/services/monitoring/quotas/L-5E141212).

Please see the AWS documentation for [Service Quotas](https://docs.aws.amazon.com/servicequotas/latest/userguide/intro.html) and [CloudWatch limits](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/cloudwatch_limits.html) for more information.
```

**Review by marcusolsson — DISMISSED** [review]
Nice! I added a couple of comments

**Comment by sunker** [comment]
Thank for feedback Marcus. I'll fix!

File: docs/sources/features/datasources/cloudwatch.md [hunk 3]
```diff
@@ -114,20 +109,62 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
 region = us-west-2
 ```
 
-## Metric Query Editor
+## Using the M  etric Query Editor
```
Comment (daniellee) [inline docs/sources/features/datasources/cloudwatch.md]: It looks ok when I build locally but why is there a space between M and etric here?

File: docs/sources/features/datasources/cloudwatch.md [hunk 4]
```diff
@@ -114,20 +109,62 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
...
+
+You need to specify a namespace, metric, at least one stat, and at least one dimension as long as `Match Exact` is activated. In the case `Match Exact` is not activated, no dimension needs to be specified.
```
Comment (daniellee) [inline docs/sources/features/datasources/cloudwatch.md]: This feels like half a sentence. I need these to do what?

File: docs/sources/features/datasources/cloudwatch.md [hunk 5]
```diff
@@ -114,20 +109,62 @@ aws_secret_access_key = dasdasdsadasdasdasdsa
...
-![](/img/docs/v43/cloudwatch_editor.png)
+By default, the search expression is defined in such a way that the queried metrics must match the defined dimension names exactly. This means that in the example below only metrics with exactly one dimension with name ‘InstanceId’ will be returned.
 
-You need to specify a namespace, metric, at least one stat, and at least one dimension.
+You can untoggle `Match Exact` to include metrics that have other dimensions defined. Disabling ‘Match Exact’ also creates a search expression even if you don’t use wildcards. We simply search for any metric that match at least the namespace, metric name, and all defined dimensions.
 
-## Metric Math
+### Multi-value template variables now use search expressions
+
+When defining dimension values based on multi-valued template variables,a search expressions is used to query for the matching metrics. This enables the use of multiple template variables in one query and also allows you to use template variables for queries that have the `Match Exact` option disabled.
```
Comment (daniellee) [inline docs/sources/features/datasources/cloudwatch.md]: - Need a space after `template variables,`
- Should be `a search expression` (no s)

File: docs/sources/features/datasources/cloudwatch.md [hunk 6]
```diff
@@ -146,35 +183,36 @@ and `dimension keys/values`.
...
-------- | -----
-*dimension_values(us-east-1,AWS/ELB,RequestCount,LoadBalancerName)* | ELB
-*dimension_values(us-east-1,AWS/ElastiCache,CPUUtilization,CacheClusterId)* | ElastiCache
-*dimension_values(us-east-1,AWS/Redshift,CPUUtilization,ClusterIdentifier)* | RedShift
-*dimension_values(us-east-1,AWS/RDS,CPUUtilization,DBInstanceIdentifier)* | RDS
-*dimension_values(us-east-1,AWS/S3,BucketSizeBytes,BucketName)* | S3
-*dimension_values(us-east-1,CWAgent,disk_used_percent,device,{"InstanceId":"$instance_id"})* | CloudWatch Agent
-*resource_arns(eu-west-1,elasticloadbalancing:loadbalancer,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})* | ELB
-*resource_arns(eu-west-1,ec2:instance,{"elasticbeanstalk:environment-name":["myApp-dev","myApp-prod"]})* | EC2
+| Query                                                                                                                         | Service          |
```
Comment (daniellee) [inline docs/sources/features/datasources/cloudwatch.md]: Formatting is also messed up here:

image

File: docs/sources/features/datasources/cloudwatch.md [hunk 7]
```diff
@@ -284,6 +336,6 @@ datasources:
       authType: keys
       defaultRegion: eu-west-2
     secureJsonData:
-      accessKey: "<your access key>"
```
Comment (daniellee) [inline docs/sources/features/datasources/cloudwatch.md]: Noted that the quotes are changed here and I think that is valid YAML but worth checking to make sure.

**Review by daniellee — CHANGES_REQUESTED** [review]
To test the docs locally:

1. Checkout [grafana/website](https://github.com/grafana/website) repo, switch to the `old-docs` branch, and pull the latest changes, then run `make docs-build`. This will take a few minutes.
2. In the Grafana repo:
    ```
   cd docs
   make watch
   ```
3. Open http://localhost:3004/features/datasources/cloudwatch/

**Review by daniellee — APPROVED** [review]

**Comment by oddlittlebird** [comment]
Dang. Got to this party just a little too late.

**Pull request merged**